scripts/core/helpers/waitForMediaToBeReady.ts

```diff
@@ -1,9 +1,17 @@
 /**
  * @param elements An array of img, audio and video elements
- * @param eventName EventListener's name
  */
-export const waitForMediaToLoad = (elements: Array<any>, eventName: string): Promise<void> => new Promise((resolve) => {
-    let itemsLeftToLoad: number = elements.length;
+export const waitForMediaToLoad = (elements: Array<Element>): Promise<void> => new Promise((resolve) => {
+    // EventListener's name
+    let eventName: string = 'load';
```

Comment:
event name depends on element type, so it's more accurate if you don't set it for the entire scope and use inline strings in conditional blocks instead.